base/version.jl

```diff
@@ -208,6 +208,14 @@ catch e
     global const VERSION = VersionNumber(0)
 end
 
+"""
+    VERSION
+
+A `VersionNumber` object describing which version of Julia is in use. For details see
+[Version Number Literals](:ref:`man-version-number-literals`).
+"""
+:VERSION
```

Comment:
Can use the unquoted form here, though perhaps rewriting the `try` block would work too. Something like:

```julia
"""
    VERSION

A `VersionNumber` object describing which version of Julia is in use. For details see
[Version Number Literals](:ref:`man-version-number-literals`)
"""
const VERSION = try
    build_number = GIT_VERSION_INFO.build_number != 0 ? "+$(GIT_VERSION_INFO.build_number)" : ""
    convert(VersionNumber, "$(VERSION_STRING)$(build_number)")
catch e
    println("while creating Base.VERSION, ignoring error $e")
    VersionNumber(0)
end
```

(Perhaps there's a reason it's written like it is?)